Source/Core/VideoBackends/Vulkan/VKRenderer.cpp

```diff
@@ -300,12 +315,12 @@ void Renderer::BindBackbuffer(const ClearColor& clear_color)
     if (res == VK_ERROR_FULL_SCREEN_EXCLUSIVE_MODE_LOST_EXT)
     {
       // The present keeps returning exclusive mode lost unless we re-create the swap chain.
-      INFO_LOG_FMT(VIDEO, "Lost exclusive fullscreen.");
+      INFO_LOG(VIDEO, "Lost exclusive fullscreen.");
```

Comment:
Please don't change those back. Mid- to long-term we want to get rid of `xxx_LOG` and rename `xxx_LOG_FMT` so everything uses `fmt`.

Reply:
Ah, sure. I haven't been keeping up to date, and assumed "_FMT" was for a "formatted variant".

Reply:
The `INFO_LOG_FMT` version means the `fmt` library is used, but it's not like there's a performance benefit to not using it; the `INFO_LOG` version uses C-style format strings (which are, IIRC, slower than fmt).